guide/payments/receive.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: guide
+title: Receiving bitcoin
+description: An introduction to how receiving bitcoin works.
+nav_order: 3
+parent: Payments
+permalink: /guide/payments/receive/
+main_classes: -no-top-padding
+# image: /assets/images/guide/payments/receive/header.svg
+---
+
+# Receiving bitcoin
+
+<!--
+
+Editor's notes
+
+This page should cover what to do when receiving bitcoin, how to share and copy addresses etc.
+
+- What addresses to support (native segwit by default)
+- How to present QR codes (allow for specifying an amount)
+- Generating a new address 
+
+-->
+
+For someone to receive bitcoin, the sender needs to know the recipient’s address. The receiver generates an address in their Bitcoin application and shares it with the sender as text, [QR Code]({{ '/guide/foundations/wallet-interoperability/#qr-codes' | relative_url }}), or a [payment link]({{ '/guide/foundations/wallet-interoperability/#payment-links' | relative_url }}). You can think of an address just as you would an invoice. A new address should be generated for every payment, as they should only be used once. This prevents the current sender from learning about previously received funds.
+
+There is other contextual information about payments that can be useful to keep or to communicate, like the names of the transacting parties or the purpose of the transaction. As the blockchain only stores the amount and addresses involved in a transaction, any additional information has to be stored in your application, or encoded in payment links when they need to be shared.
+
+Let's look at our options when receivers want to share other details with the sender, or store them with the payment request for their own record keeping.
+
+<!-- 
+Update glossary
+
+Each address that a Bitcoin application generates has an accompaniying private key. This private key allows the receiver to unlock the bitcoin that is sent to the address when the receiver chooses to spend it.
+
+Although addresses are not a problem to share publically, their accompanying private keys which typically generated from a single recovery phrase must be kept secure and private.
+-->
+
+## Creating an invoice
```

Comment:
@GBKS @pavlenex thoughts on referring to generating an address and adding metadata to it as an invoice? I previously called it payment requests, and @danielnordh prefers that as it is less corporate/businessy.

Reply:
I find invoice to be more of business term than payment request (thinking about which terms I'd use with friends vs. business purposes). So my vote would be for payment request (despite it being longer).

Reply:
I prefer an invoice, easier to translate, the payment request is not a regular term in my opinion.

Reply:
When we start talking about LN, it's all about invoices there, and some of the additional data is going to be more built-in.

Invoice
- https://en.bitcoin.it/wiki/Invoice_address
- https://github.com/lightningnetwork/lightning-rfc/blob/master/11-payment-encoding.md

#### Payment Request
- https://en.bitcoin.it/wiki/Payment_Request

👆 this is old deprecated BIP but still has some good ideas around signed messaged that [will probably be rehashed](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-March/018560.html).

Reply:
I commented a bit more on this part in the [doc](https://docs.google.com/document/d/1PMgMIZ1fkC3SqJKWQBOrmJxefjmZhktqbSPZfzabmrE/edit?usp=sharing)

My feedback is basically:

- If 'invoice' is not a defined standard in Bitcoin, let's not push for it as it doesn't really make sense for end users (maybe for merchants, but that's not what we are tackling here)
- We don't not need to talk about this as creating an 'object', so instead of saying 'Create an invoice/payment request', we can simply say 'Requesting bitcoin' as a subtitle. For the user it's an action, not an object.
- IF the user is asked/able to enter additional information other than the address, it should be clear to them if that information is SHARED with the person they want to request bitcoin from, or not

Reply:
Good point about this object-creation idea. I prefer the simpler language that avoids that concept and focuses on the action.

Reply:
I wouldn't dismiss the object creation idea - as sending bitcoin is very much action oriented flow, the act of requesting bitcoin has artefacts that the designer would need to consider in other areas of the application and would benefit from understanding that object and its properties. So I think in this case it would be very well suited.

Since this comment though, the state of the page has gone towards something which is more action oriented, I'll probably bring it up at another point though after the chapter complete for v1.